src/backend/src/players/players.controller.ts

```diff
@@ -13,6 +13,9 @@ export class PlayersController {
         'Create Player requires emotionId and userId',
       );
     }
+    if (emotionId != 0 && emotionId != 1) {
+      throw new BadRequestException('EmotionId has to be either 0 or 1');
+    }
```

Comment:
This seems brittle. What happens when they add more emotions? We keep adding to this list? Is there a way to check that it's a member of the enum